design/common.blocks/link/_theme/link_theme_islands.styl

```diff
@@ -18,6 +18,21 @@
         {
             color: #070;
         }
+
+        &_ghost
+        {
+            color: #999;
+        }
+
+        &_black
+        {
+            color: #000;
+        }
+
+        &_strong
```

Comment:
But do we need them? It's trivial to implement them as well as links with other colours or font styles.